tests/zenko_tests/node_tests/smoke_tests/vault_admin_and_IAM_API_tests/iamApi.js

```diff
@@ -0,0 +1,43 @@
+const async = require('async');
+
+const VaultClient = require('./VaultClient');
+
+const clientAdmin = VaultClient.getAdminClient();
+const accountName = 'iam-api-test-account';
+const accountInfo = {
+    email: `${accountName}@test.com`,
+    password: 'test',
+};
+const externalAccessKey = 'DZMMJUPWIUK8IWXRP0HQ';
+const externalSecretKey = 'iTuJdlidzrLipymvAGrLP66Yxghl4NQxLZR3cLlu';
+const userName = 'iam-api-test-user'
+
+describe('IAM users: ', () => {
+    let iamAccountClient = null;
+
+    beforeEach(done => async.series([
+        next => clientAdmin.createAccount(accountName, accountInfo, next),
+        next => clientAdmin.generateAccountAccessKey(accountName, next, { externalAccessKey, externalSecretKey }),
+    ], () => {
+        iamAccountClient = VaultClient.getIamClient(externalAccessKey, externalSecretKey);
+        return done();
+    }));
+
+    afterEach(done => iamAccountClient.deleteUser({ UserName: userName},
+        () => clientAdmin.deleteAccount(accountName, done)));
```

Comment:
Same thing for the `delete user` call, we want to know if it fails.
For `should be able to create, list and delete a user` test, in your "after each" step, you might probably see an `NoSuchEntity` error.
To fix that, you might have to add another `describe` level with `afterEach(done => iamAccountClient.deleteUser({ UserName: userName}, done));` for `'should be able to create a user'` and `'should be able to generate keys for a user'`

Reply:
Changed in review commit: [`b875a09` (#1207)](https://github.com/scality/Zenko/pull/1207/commits/b875a09c7268760a05fe9973830a9119b606b17b)